extensions/amp-form/0.1/test-e2e/test-ssr.js

```diff
@@ -0,0 +1,50 @@
+/*e
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Amp-form test for standalone and in an SSR enabled viewer. Verifies that
+ * the rendered output is the same if the response from the REST endpoint is
+ * the same as from the SSR endpoint.
+ */
+
+describes.endtoend.only('AMP form server side rendered templates', {
```

Comment:
`.only`?

Reply:
right. that should've been caught by lint. removed.